### ISSUE TITLE
Adding some examples

### ISSUE THREAD

**Issue by jianjunz** [WICG/web-codecs#78]
Examples would be a great start for developers to try web codecs. Perhaps we could add a new section in the specification, and move examples from the explainer.

**Comment by lincolnneu** [comment]
Yes please add real examples using webcodecs. I'm looking for getting the frame number of the frame displayed currently on the <video/> node or equivalent.

**Comment by chcunningham** [comment]
Demo samples are now added here
https://w3c.github.io/webcodecs/samples

I've updated the readme with a link as well. 

Still, the spec itself may benefit from some example code, so I'll leave this open to track that editorial work.
